.gitignore

```diff
@@ -35,3 +35,5 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+.idea/*

```

Comment:
i guess 🙄 
we do this in the buildiumcode repo too, but i think it really belongs in a global gitignore file on dev computers

the only argument that justifies this is "but then every developer has to update their gitignore file, and this is faster"

but, now that this is "open source" i can counter with "we shouldn't try to account for every developers ide and there's nothing special about intellij or whatever is generating this file" 😼